melos.yaml

```diff
@@ -16,7 +17,3 @@ scripts:
     exec: dart run $MELOS_ROOT_PATH/tool/expose_version.dart
   generate:
     run: cd mobx_examples && dart run build_runner watch --delete-conflicting-outputs
-
```

Comment:
This is required to work locally. We should not remove this

Reply:
this options has been removed in melos 3.1.1, isn't it?
https://github.com/invertase/melos/blob/melos-v3.1.1/docs/guides/migrations.mdx#L52

Reply:
Awesome .. didn't know that ...